package.json

```diff
@@ -1,42 +1,40 @@
 {
```

Comment:
this is just a side affect of `npm install [dep] --save`, so we should prob commit the new white-space